tools/schemas/src/json_schema.rs

```diff
@@ -195,18 +232,28 @@ fn extract_refs(schema: &Schema) -> Vec<Ref> {
         refs.extend(extract_refs(item_schema));
     }
 
-    if let Some(defs) = &schema.defs {
-        for def in defs.values() {
-            refs.extend(extract_refs(def));
-        }
-    }
+    // if let Some(defs) = &schema.defs {
+    //     for def in defs.values() {
+    //         refs.extend(extract_refs(def));
+    //     }
+    // }
```

Comment:
Commented code

Q: do we need it?

Reply:
thanks for noticing ... not sure how it happened, but now it's uncommented